dvwa/compose.yml

```diff
@@ -0,0 +1,26 @@
+# Originally sourced (and modified) from the compose file at https://github.com/digininja/DVWA
+
+volumes:
+  dvwa:
+
+services:
+  dvwa:
+    image: ghcr.io/digininja/dvwa:latest
```

Comment:
Don't think it's worth including a pull policy on both these services (as the only impact will be on the trainers and will likely cause more problems that it'll fix).